client/src/featureform/serving.py

```diff
@@ -1020,6 +1035,45 @@ def dataframe(self) -> pd.DataFrame:
             self._dataframe.rename(columns={cols.label: "label"}, inplace=True)
             return self._dataframe
 
+    def get_file_format_and_location(self, location):
+        if location.endswith(".csv"):
```

Comment:
we have a whole `FileFormat` and `FilePrefix` class for a lot of this, can we use those here?